WordPress/Classes/ViewRelated/Site Creation/FinalAssembly/SiteAssemblyContentView.swift

```diff
@@ -111,14 +153,46 @@ final class SiteAssemblyContentView: UIView {
             label.textColor = .textSubtle
             label.textAlignment = .center
 
-            let statusText = NSLocalizedString("We’re creating your new site.",
+            let statusText = NSLocalizedString("Your site will be ready shortly",
                                                comment: "User-facing string, presented to reflect that site assembly is underway.")
             label.text = statusText
             label.accessibilityLabel = statusText
 
             return label
         }()
 
+        self.statusImageView = {
+            let image = UIImage(named: "site-creation-loading")
+            let imageView = UIImageView(image: image)
+
+            return imageView
+        }()
+
+        self.statusMessageRotatingView = {
+            //The rotating message view will automatically use the localized string based
+            //on the message
+            let statusMessages = [
+                "Grabbing site URL",
```

Comment:
Can you please add those strings under `NSLocalizedString` and add a description?

Those strings will be extracted and internationalized for various idioms.

Reply:
So this automatically happens automatically here: https://github.com/wordpress-mobile/WordPress-iOS/blob/d22641a938a15d45d1ad2172f14949e52ce9268f/WordPress/Classes/ViewRelated/Site%20Creation/FinalAssembly/SiteCreationRotatingMessageView.swift#L141 itself. 

Should I remove this line from the message view and move it to the `SiteAssemblyContentView`?

Reply:
Hmm, I'm not sure if our script will be able to pick up those new strings for translation.

Let's check! @jkmassel @loremattei new strings must be under `NSLocalizedString` to be extracted or that should work?

Reply:
@emilylaguna I just confirmed with @jkmassel and we need the strings to be under `NSLocalizedString`. Otherwise, they'll be ignored by the script that extracts them.